tests/spread/integration/aspnetcore-runtime-9.0/task.yaml

```diff
@@ -0,0 +1,25 @@
+summary: Integration tests for ASP.NET 9 Runtime
+
+execute: |
+  # install slices
+  rootfs="$(install-slices aspnetcore-runtime-9.0_libs base-files_base)"
+  mv "${rootfs}"/usr/lib/dotnet/dotnet9 "${rootfs}"/usr/lib/dotnet/dotnet
+  ln -s ../lib/dotnet/dotnet "${rootfs}"/usr/bin/dotnet
+
+  # smoking test the .NET runtime
+  chroot "${rootfs}" /usr/bin/dotnet --info
+
+  # preparing the test data
+  apt update && apt install -y dotnet-sdk-9.0
+  cp -r web_helloworld "${rootfs}"/web_helloworld
+  dotnet publish "${rootfs}"/web_helloworld/Hello.csproj --no-self-contained
+
+  mkdir -p "${rootfs}"/proc
+  mount --bind /proc "${rootfs}"/proc
+
+  # test the helloworld web app
+  chroot "${rootfs}" dotnet /app_helloworld/bin/Release/net9.0/Hello.dll --urls="http://0.0.0.0:5108" &
+  sleep 5
```

Comment:
I would wait for the port to open like we do here. Worst this fails due to race condition, best case we speed up the test.
Here is an example with fuser you can paste in.
https://github.com/canonical/chisel-releases/blob/ubuntu-24.10/tests/spread/integration/git/task.yaml#L98